config/sync/migrate_plus.migration.node_news.yml

```diff
@@ -57,9 +56,38 @@ process:
       plugin: get
       source: field_summary
   field_teaser: field_teaser
+  field_photo:
+    -
+      plugin: sub_process
+      source: field_photo
+      process:
+        target_id:
+          -
+            plugin: d7_file_lookup
```

Comment:
Here's a good example of where we need this plugin, over migration_lookup, because our fid value is a D7 file id... and we need to look up the D9 file id instead. This plugin does the hard work of doing that for us.